paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSheetScreenshotTest.kt

```diff
@@ -191,7 +187,6 @@ internal class CustomerSheetScreenshotTest {
                     title = "Screenshot testing",
                     paymentSelection = PaymentSelection.GooglePay,
                     isGooglePayEnabled = true,
-                    primaryButtonLabel = "Continue",
```

Comment:
`SelectPaymentMethod` always had `Confirm` as its text but we just used `Continue` in screenshot tests.